lib/src/sync_worker_loop.dart

```diff
@@ -19,6 +19,8 @@ abstract class SyncWorkerConnection {
 }
 
 /// Persistent Bazel worker loop.
+///
+/// Extend this class and implement the `performRequest` method.
 abstract class SyncWorkerLoop {
```

Comment:
Personally I would favor converting analyzer_cli to async and just keeping the async version for the sake of having all code paths used in practice from the outset.  But I won't press the issue.